lib/src/local/request.rs

```diff
@@ -189,6 +189,29 @@ impl<'c> LocalRequest<'c> {
         self
     }
 
+    /// Add a [private cookie] to this request.
+    ///
+    /// [private cookie]: /rocket/http/enum.Cookies.html#private-cookies
+    ///
+    /// # Examples
+    ///
+    /// Add `user_id` as a private cookie:
+    ///
+    /// ```rust
+    /// use rocket::local::Client;
+    /// use rocket::http::Cookie;
+    ///
+    /// let client = Client::new(rocket::ignite()).unwrap();
+    /// # #[allow(unused_variables)]
+    /// let req = client.get("/")
```

Comment:
Can we fit this and the next line in 80 cols? If so, let's merge them.

Reply:
No, we can't. It's 81 :(